tuf/ngclient/_internal/proxy.py

```diff
@@ -0,0 +1,101 @@
+# Copyright New York University and the TUF contributors
+# SPDX-License-Identifier: MIT OR Apache-2.0
+
+"""Proxy environment variable handling with Urllib3"""
+
+from __future__ import annotations
+
+from typing import Any
+from urllib.request import getproxies
```

Comment:
we could implement this method ourselves as well but I tried to keep this minimal